### DISCUSSION TITLE
Basic form builder example

### DISCUSSION THREAD

**Discussion by fellz** [vueform/vueform#289] [Questions]
Hello
I found this example on the site but i think it's kind of outdated.
[https://vueform.com/examples/basics-form-builder](https://vueform.com/examples/basics-form-builder)
Is there a version for Composition API ? Should it us mixins ?
@adamberecz

**Comment by adamberecz** [comment]
It's not outdated, just that specific example is made with Options API. There are [different ways of using Vueform](https://vueform.com/docs/rendering-forms) you might tailor the example to your choice.
